tools/dependency/release_dates.py

```diff
@@ -211,7 +211,7 @@ def get_tagged_release_date(repo, metadata_version, github_release):
         latest = ''
         print(f'GithubException {repo.name}: {err.data} {err.status} while getting latest release.')
 
-    if latest and github_release.version <= latest.tag_name:
+    if latest:
```

Comment:
@moderation im not sure what the logic here is - i would have thought it always wants to return the date of the github_release